events/elections/2021/README.md

```diff
@@ -45,46 +45,79 @@ Eligibility for voting in 2021 is defined as:
+to a consensus on who should serve on the committee.
+
+### Voter Exemption
+
+We *explicitly* believe that the above heuristic will be inaccurate
+and not represent the entire community. Thus we provide the form
+for those who have contributed to the project but may not meet the above
+criteria. Acceptance of a form submission will be defined by a simple
+majority vote, and the criteria used during this process will be used to
+help refine further elections.
 
 If you otherwise qualify to vote but have not yet applied for Org Membership,
 then please [request an exception][voter exemption form] (and please apply for
 Org Membership as well).
 
-Corporate affiliation is applied after the election. If an organization finds
-itself with too many representatives it is up to those individuals to come
-to a consensus on who should serve on the committee.
+Only contributions to projects and artifacts that fall under Steering
+Committee's governance will be considered for voter exemption.
+
+Examples of contributions that would be considered:
+* Slack admins who are not active in GitHub
+* Code of Conduct Committee members whose actions are private by default
+
+Examples of contributions that would NOT be considered:
+* Contributions to ecosystem projects and products
+* Organizing meetups or podcasts
 
 ### Schedule
 
+<!-- While finalizing the dates in the schedule, ensure that:
+- The Steering Committee and candidate Q+A occurs at a public SC meeting
+  (usually a Monday).
+- Dealine to submit voter exception forms and request a
+  replacement ballot is ~3 days before voting closes.
+- Private announcement of results to SC members is at least ~2 days
+  before private announcement to all candidates.
+- The interval between private announcement to all candidates and the
+  public announcement is a weekend.
+-->
+
 | Date         | Event                    |
 | ------------ | ------------------------ |
-| August XX    | Announcement of Election and publication of Voters.md |
+| July 1       | Steering Committee selects Election Committee |
+| August XX    | Announcement of Election and publication of voters.md |
 | August XX    | Steering Committee Meeting with Q+A with the candidates and community |
-| September XX | All candidate bios and voting exception forms due by 0000 UTC (5pm PST) |
+| September XX | All candidate bios due by 0000 UTC (5pm PST) |
 | ~1 week      | Election prep week (voters.md validation and CIVS setup and testing)
 | September XX | Election Begins via email ballots |
-| October XX   | Deadline to request a replacement ballot |
+| October XX   | Deadline to submit voter exception forms and request a replacement ballot |
 | October XX   | Election Closes by 0000 UTC (5pm PST) |
-| October XX   | Announcement of Results at Public Steering Committee meeting |
+| October XX   | Private announcement of Results to SC members not up for election |
+| October XX   | Private announcement of Results to all candidates |
```

Comment:
remind me why these two steps were split? communicating with a subset of steering might be tricky (I guess you'd have to drop down to individual emails to avoid sending to the private-steering list?)

Reply:
No good reason, it was just how we did it in the past.

Reply:
> remind me why these two steps were split?

It was also done to make sure any changes needed to accommodate https://github.com/kubernetes/community/issues/5092 were discussed before the results were disclosed to all candidates.

Once we figure out how to go ahead with https://github.com/kubernetes/community/issues/5092, we can probably remove this split.